shelley/chain-and-ledger/formal-spec/chain.tex

```diff
@@ -1240,40 +1241,52 @@ \subsection{Verifiable Random Function}
 
 \begin{figure}
   \emph{VRF helper function}
+  \begin{align*}
+      & \fun{poolIDfromBHBody} \in \BHBody \to \KeyHash_{pool} \\
```

Comment:
The nomenclature is maybe a bit confusing here, since the issuer is not necessarily a pool. So I would go with `issuerID` or similar

Reply:
that's a good point, thank you! I would like to keep the name the same between the formal spec and the code, and I know that `poolIDfromBHBody` is being used already downstream (db sync uses it), but I can just give folks a heads up.

Reply:
I've deprecated `poolIDfromBHBody` and made an identical `issuerIDfromBHBody`.